lib/fetch/util.js

```diff
@@ -837,7 +837,14 @@ function fullyReadBody (body, processBody, processBodyError) {
   }
 
   // 5. Read all bytes from reader, given successSteps and errorSteps.
-  readAllBytes(reader, successSteps, errorSteps)
+  try {
+    const result = await readAllBytes(reader)
+    await true
```

Comment:
I don't think these awaits are necessary?

Reply:
it's to follow the spec by adding in a microtick https://github.com/nodejs/undici/issues/2212#issuecomment-1680859014

Reply:
Isn't there already a micro tick from the await above?

Reply:
makes sense, removed them